src/js/visuals/visualization.js

```diff
@@ -257,12 +255,11 @@ var Visualization = Backbone.View.extend({
   myResize: function() {
     if (!this.paper) { return; }
 
-    var smaller = 1;
     var el = this.el;
 
     var elSize = el.getBoundingClientRect();
-    var width = elSize.width - smaller;
```

Comment:
did this `1` not matter here?

Reply:
If `elSize.width === 0` it will throw error (set width to -1).

We can catch `width < 1`